js/layers/scenario_layer/main.js

```diff
@@ -356,6 +356,16 @@ ubsApp.openPopup = function(config) {
    ubsApp.startHelp("generalPopUp");
 }
 
+ubsApp.callServerClosePopup = function(){
+	console.log("close pop up clicked");
```

Comment:
Once app is tested in local. Please remove the loggers. We don't want to end up in bunch of loggers on console while game is being played.

Merging for now, but please remove it in your next PR.